lib/kdump_utils.pm

```diff
@@ -278,7 +279,7 @@ sub activate_kdump_cli {
         } else {
             $status = script_run('yast kdump show 2>&1 | grep "Kdump is disabled"', 180);
         }
-        return if ($status and !get_var('CRASH_MEMORY') and !get_var('FADUMP'));
+        return if ($status and !get_var('CRASH_MEMORY') and !get_var('FADUMP') and !get_var('KDUMP_SAVEDIR'));
```

Comment:
I believe that all troubles were caused by '/etc/exports' and we don't need this obscure extension.

Reply:
OK, check is ongoing:
https://openqa.suse.de/tests/18139995
https://openqa.suse.de/tests/18139996
https://openqa.suse.de/tests/18139994

Reply:
https://openqa.suse.de/tests/18139996#step/kdump_over_nfs/464
looks indeed good